app/views/message_mailer/new_message.html.erb

```diff
@@ -1,3 +1,7 @@
 <p><%= @sender %> sent you <%= link_to "a message", notification_url(@notification.record) %>:</p>
 
-<%== html_truncate_message_body(@body) %>
+<%== @body %>
+
+<% if Feature.enabled?(:reply_via_email, user: @recipient) %>
+  <p><small>You can continue the conversation by replying to this email.</small></p>
```

Comment:
👋 Saw the livestream…

In case you did want to go the `<hr>` route, the issue with `margin` was that you were comma-separating the values, which is invalid:

```diff
-<hr style="margin: 20px, 0;">
+<hr style="margin: 20px 0;">
```

So…

```suggestion
  <hr style="margin: 20px 0;">
  
  <p>You can continue the conversation by replying to this email.</p>
```

Reply:
Bah, of course! Tailwind CSS has ruined me. 😆